app/src/androidTest/java/com/duckduckgo/app/browser/BrowserTabViewModelTest.kt

```diff
@@ -459,7 +462,8 @@ class BrowserTabViewModelTest {
             trackingParameters = mockTrackingParameters,
             voiceSearchAvailability = voiceSearchAvailability,
             voiceSearchPixelLogger = voiceSearchPixelLogger,
-            settingsDataStore = mockSettingsDataStore
+            settingsDataStore = mockSettingsDataStore,
+            autofillStore = mockAutofillStore
```

Comment:
We can more tests here for the new functions we added.

Reply:
Deferring, captured in https://app.asana.com/0/0/1202592915624853/f